src/converters/bool-conditional-converters.js

```diff
@@ -14,17 +34,54 @@ function parseArgs(trueVal, falseVal, input, matchString) {
 
 module.exports = {
     /**
-     * Converts a 'truthy' values to true and 'falsy' values to false
+     * Convert 'truthy' values to true and 'falsy' values to false.
+     *
+     * **Example**
+     *
+     *      <!-- displays "true" or "false" -->
+     *      <span data-f-bind="sampleVar | toBool"></span>
+     * 
      * @param {Any} value
      * @return {Boolean}
      */
     toBool: function (value) {
         return !!value;
     },
+
+    /**
+     * Convert the input to a new value (for example, some text), based on whether it is true or false.
+     *
+     * **Example**
+     *
+     *      <div>
+     *          <span data-f-bind="sampleVar | ifTrue('yes! please move forward', 'not ready to proceed')"></span> 
+     *          <span data-f-bind="sampleVar | ifTrue('yes! please move forward')"></span>
+     *      </div>
+     *
+     * @param {Number} trueVal The value to display if the input is true.
+     * @param {Number} falseVal (Optional) The value to display if the input is false. If not included, returns `false`.
+     * @param {Any} input (Optional) The input to test. If not included, the output of the previous argument is used.
+     * @return {Any} If input is true, returns trueVal. If input is false, returns falseVal if provided, or `false`.
```

Comment:
This was supposed to return the original value if not true and no falseval specified; is that not happening?